coffea/nanoevents/mapping/uproot.py

```diff
@@ -67,7 +67,10 @@ def _extract_base_form(cls, tree, iteritems_options={}):
                 form["content"]["parameters"] = {"__doc__": branch.title}
             elif (
                 form["class"].startswith("ListOffset")
-                and form["content"]["class"].startswith("ListOffset")
+                and (
+                    form["content"]["class"].startswith("ListOffset")
+                    or form["content"]["class"] == "RegularArray"
+                )
```

Comment:
This addition seems to allow for getting `Edges[4]`, `Tau[5]`, and `FracPt[5]` to work correctly(?) maybe? At least it seems to be ok with uproot.